src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -162,6 +166,8 @@ class VtolAttitudeControl
 	struct vehicle_local_position_s		_local_pos;
 	struct airspeed_s 					_airspeed;			// airspeed
 	struct battery_status_s 			_batt_status; 		// battery status
+	struct vehicle_command_s			_vehicle_cmd;
+	struct vehicle_command_s			_vehicle_transition_cmd; // stores transition commands only
```

Comment:
I don't understand why there are two vehicle_command_s, it seems all the same, or am I not getting it?